model/src/main/proto/thumbnail.proto

```diff
@@ -38,65 +38,3 @@ enum LessonThumbnailGraphic {
   // Corresponds to an in-app image of a person next to a pie chart.
   PERSON_WITH_PIE_CHART = 6;
 }
-
-// Represents the thumbnail that should be rendered for a skill.
-message SkillThumbnail {
-  // The graphic corresponding to this thumbnail.
-  SkillThumbnailGraphic thumbnail_graphic = 1;
-}
-
-// Corresponds to different types of thumbnails that may be used by skills in a topic. These currently only
-// correspond to in-app thumbnails, but may later indicate that a skill has a downloadable thumbnail to use, instead.
-enum SkillThumbnailGraphic {
-  // Corresponds to Identifying the parts of a fraction skill. Also, acts as default thumbnail for skill.
-  IDENTIFYING_THE_PARTS_OF_A_FRACTION = 0;
```

Comment:
Add all these to the `Lesson thumbnail` so that you can link them in domain layer and finally in app module

Reply:
Done :)